backend/src/services/__init__.py

```diff
@@ -0,0 +1,4 @@
+from NotificationService import Notification, NotificationService
+
+Notification = Notification
```

Comment:
Ymm, I'm not sure if this `Notification = Notification` even makes a difference. Reexports don't work like that